CMakeLists.txt

```diff
@@ -19,6 +19,14 @@ set(CMAKE_Fortran_MODULE_DIRECTORY ${PROJECT_BINARY_DIR}/modules)
 # Define GNU standard installation directories
 include(GNUInstallDirs)
 
+#find_package(BLAS REQUIRED)
```

Comment:
Why is this section commented out? Is it redundant? If so, then remove, otherwise we should be able to programmatically choose between the two options

Reply:
I've sorted this